src/character.cpp

```diff
@@ -443,6 +443,7 @@ Character::Character() :
     healthy_calories = 55'000'000;
     stored_calories = healthy_calories;
     initialize_stomach_contents();
+    set_cardio_acc( get_bmr() / 2 );
```

Comment:
Is there an activity level at this point, and if so, what is it? That will affect what get_bmr() returns.

Reply:
```suggestion
    set_cardio_acc( base_bmr() / 2 );
```

Reply:
it'd be 0